Lib/importlib/_bootstrap_external.py

```diff
@@ -692,6 +692,14 @@ class WindowsRegistryFinder:
         '\\Modules\\{fullname}\\Debug')
     DEBUG_BUILD = (_MS_WINDOWS and '_d.pyd' in EXTENSION_SUFFIXES)
 
+    def __init__(self):
+        import warnings
+        warnings.warn('importlib.machinery.WindowsRegistryFinder is '
+                      'deprecated. Use site configuration instead. '
+                      'Future versions of Python may not enable this '
+                      'finder by default.',
```

Comment:
```suggestion
        warnings.warn("importlib.machinery.WindowsRegistryFinder is deprecated and will be removed in a future version of Python."                           
        "Please use site configuration instead."
```

Reply:
I used the same wording from the docs. I don't actually know if the plan is to eventually remove it so I wanted to stay on the safe side..